app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/multiple_disposals_task_list.scala.html

```diff
@@ -55,9 +55,13 @@
 }
 
 @ytdLiabilityState = @{
-    draftReturn.yearToDateLiabilityAnswers.fold[TaskListStatus](TaskListStatus.ToDo) {
-        case c: CalculatedYTDAnswers    => c.fold(_ => TaskListStatus.InProgress, _ => TaskListStatus.Complete)
-        case n: NonCalculatedYTDAnswers => n.fold(_ => TaskListStatus.InProgress, _ => TaskListStatus.Complete)
+    exemptionsAndLossesState match {
```

Comment:
is there a unit test for this? This looks like new behaviour

Reply:
we might not have unit tests for the multiple disposals task list - it'd be nice to have them as part of this ticket